pallets/collator-assignment/src/lib.rs

```diff
@@ -177,10 +182,43 @@ pub mod pallet {
                 let mut rng: ChaCha20Rng = SeedableRng::from_seed(random_seed);
                 collators.shuffle(&mut rng);
                 container_chain_ids.shuffle(&mut rng);
+                parathreads.shuffle(&mut rng);
             }
 
             // We read current assigned collators
             let old_assigned = Self::read_assigned_collators();
+            // Initialize list of container chains as `[orchestrator, container1, container2, parathread1, parathread2]`.
+            // The order means priority: the first chain in the list will be the first one to get assigned collators.
+            // Chains will not be assigned less than `min_collators`, except the orchestrator chain.
+            // First all chains will be assigned `min_collators`, and then the first one will be assigned up to `max`,
+            // then the second one, and so on.
+            let mut container_chains = vec![ContainerChain {
+                para_id: T::SelfParaId::get(),
+                min_collators: T::HostConfiguration::min_collators_for_orchestrator(
+                    target_session_index,
+                ),
+                max_collators: T::HostConfiguration::max_collators_for_orchestrator(
+                    target_session_index,
+                ),
+            }];
+            let collators_per_container =
+                T::HostConfiguration::collators_per_container(target_session_index);
+            for para_id in &container_chain_ids {
```

Comment:
Can you put a TODO here indicating that this is provisional and that the order should be done based on tipping in the future?

Reply:
Sure! I imagine that the priority will be done instead of shuffling here, so the rest of the algorithm should not change when we add tipping.